### PR TITLE
add new recipe for residualvm

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

File: games-engines/residualvm/residualvm-0.2.1.recipe [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+	lib:libSDL$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libbz2$secondaryArchSuffix
+	devel:libfreetype$secondaryArchSuffix
+	devel:libgl$secondaryArchSuffix
+	devel:libGL$secondaryArchSuffix
```
Comment (korli) [inline games-engines/residualvm/residualvm-0.2.1.recipe]: Only one devel:libgl is needed.

File: games-engines/residualvm/residualvm-0.2.1.recipe [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+CHECKSUM_SHA256="1f3ff363d00f799e38ee08f985b65e07134782f63ff065a5bec68fcab4d578e4"
+
+ARCHITECTURES="!x86_gcc2 x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	ResidualVM$secondaryArchSuffix = $portVersion
+	app:ResidualVM$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
```
Comment (KapiX) [inline games-engines/residualvm/residualvm-0.2.1.recipe]: You are missing `lib:libbz2` and `lib:libglu`.

Comment (Begasus) [inline games-engines/residualvm/residualvm-0.2.1.recipe]: both have been added

File: games-engines/residualvm/residualvm-0.2.1.recipe [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://www.residualvm.org/downloads/release/$portVersion/residualvm-$portVersion-sources.tar.gz"
+CHECKSUM_SHA256="1f3ff363d00f799e38ee08f985b65e07134782f63ff065a5bec68fcab4d578e4"
+
+ARCHITECTURES="!x86_gcc2 x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	ResidualVM$secondaryArchSuffix = $portVersion
```
Comment (diversys) [inline games-engines/residualvm/residualvm-0.2.1.recipe]: Should be residualvm$secondaryArchSuffix = $portVersion

File: games-engines/residualvm/residualvm-0.2.1.recipe [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+REVISION="1"
+SOURCE_URI="http://www.residualvm.org/downloads/release/$portVersion/residualvm-$portVersion-sources.tar.gz"
+CHECKSUM_SHA256="1f3ff363d00f799e38ee08f985b65e07134782f63ff065a5bec68fcab4d578e4"
+
+ARCHITECTURES="!x86_gcc2 x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	residualvm$secondaryArchSuffix = $portVersion
+	app:residualvm$secondaryArchSuffix = $portVersion
```
Comment (diversys) [inline games-engines/residualvm/residualvm-0.2.1.recipe]: Should be app:ResidualVM$secondaryArchSuffix = $portVersion

**Pull request merged**